scripts/doc-generation/generate-docs.sh

```diff
@@ -37,10 +37,8 @@ function finish { >&2 echo -e "\033[1m\033[32m✔ $@\033[0m"; }
 
 SCRIPT_DIR=`dirname "$0"`
 
-# Search for Jazzy and install documentation dependencies if necessary.
-if [ -z `which jazzy` ]; then
-    $SCRIPT_DIR/install-documentation-dependencies.sh
-fi
+# Install documentation dependencies
+$SCRIPT_DIR/install-documentation-dependencies.sh
```

Comment:
`install-documentation-dependencies.sh` contains appropriate checks for whether jazzy already exists, so I updated this to just invoke it unconditionally.